README.md

```diff
@@ -2,20 +2,31 @@
 
 Starter project for a chess-based TDD exercise.
 
-## Running the application
+## Getting the application up and running
 
-To run the application, first ensure that you have installed Poetry on your system. Install
-dependencies using the command ``poetry install``.
+### Installing dependencies
+First, let's ensure that you have installed python and pip on your system. 
 
-To run the application, use the command ``poetry run flask run`` and navigate to http://localhost:5000. You can reset the board at any time by restarting the server.
+- Run `python --version` and you should see a version number. If not, you may need to install Python on your machine. On ACG, this should be already available - ensure you're using the image specified in the instructions.
+- Run `pip --version` and you should also see a version number. If not, you may need to install pip on your machine. On ACG, you can do `sudo apt install python3-pip` to install pip.
+
+After that, install dependencies using the command `pip install -r requirements.txt`.
+
+### Running the application
+
+- To run the application, use the command ``python -m flask run``.
+- If you are using a local machine, you can now navigate to `http://localhost:5000` in your browser to see the chess board.
+- If you are running on ACG, you will need to tunnel the port. You can tunnel the port to your local machine if you've read up and know how to do it (use SSH tunneling), or you can use a service like [pinggy.io](https://pinggy.io/) to tunnel the port to a public URL.
```

Comment:
I really thought this would be easier, but worth noting that learners can expose their public IP on ACG:
* First, open the port you want to share: `sudo ufw allow 8000`
  * [Noting that the default 5000 isn't allowed](https://help.pluralsight.com/hc/en-us/articles/24425583760532-Hands-on-playground-FAQ)
* Then run the app on an exposed host: `python -m flask run --host 0.0.0.0 --port 8000`
* Then visit the public address (e.g. I've left mine running on <http://18.133.240.91:8000/> for now)

I can't verbalise why I always find tunnelling a slightly confusing experience, but I instinctively lean away if we can find an alternative

Reply:
Omg, thank you - that article is the missing piece I needed! I'd come to the conclusion that there was just a blanket firewall in front of all ACG instances, since I tried disabling ufw entirely and experimenting with different ports with flask and not managing to connect to it remotely. I also prefer not tunnelling just because there are fewer moving parts, I just couldn't get it to work.